src/codecs/TypingNotification.ts

```diff
@@ -0,0 +1,49 @@
+import { ContentTypeId, ContentCodec, EncodedContent } from '../MessageContent'
+
+// xmtp.org/typingNotification
+//
+// This content type is used for typing notifications
+export const ContentTypeTypingNotification = new ContentTypeId({
```

Comment:
We should include some strongly worded comments here that this content type should only be used on ephemeral topics, if we can't enforce or warn at runtime. 

Otherwise someone perusing the code might be inclined to use the two features in this PR separately.

Reply:
I don't think there's currently a way to check at runtime. Thoughts on adding a field to `ContentCodec` like `ephemeralOnly`? That'd give us something we could check.